desktop/assets/react_example.coffee

```diff
@@ -1,10 +1,7 @@
 require('backbone').$ = $
 
 routes =
-  '''
-  /react-example/
-  ''': require('../apps/react_example/client.js').default()
+  '/react-example': require('../apps/react_example/client.js').default()
 
-for paths, init of routes
-  for path in paths.split('\n')
-    $(init) if location.pathname.match path
+for path, init of routes
+  $(init) if location.pathname.match path
```

Comment:
Thanks for looking at this as this file was a cut and paste job. Let me know if this doesn't fix it and I can dig in.